doc/Random.xml

```diff
@@ -33,6 +33,9 @@
 			<argument index="2" name="weights" type="PoolIntArray" default="null" />
 			<argument index="3" name="cumulative" type="bool" default="false" />
 			<description>
+				Returns an [Array] of size [code]count[/code] of random elements picked from [code]from[/code] using weights from either [code]weights[/code] or values of [code]from[/code] if [code]from[/code] is a [Dictionary] (and if [code]weights[/codes] is empty).
+				All elements of [code]weights[/code] must be non-negative integers (non-decreasing if cumulative) and of them should not be zero.
```

Comment:
Perhaps "positive" could already imply "non-zero"?

```suggestion
				All elements of [code]weights[/code] must be positive integers, and must not contain zero values. The order of integers should be non-decreasing if [code]cumulative[/code] is [code]true[/code].
```

Reply:
No, by 'non-negative' I meant positive and zero.
The array could have some elements as zero but not all. (I missed writing 'all')